pygmt/tests/test_info.py

```diff
@@ -43,7 +43,6 @@ def test_info_dataframe():
 
 
 @pytest.mark.xfail(
-    condition=gmt_version <= Version("6.1.1"),
```

Comment:
At the top of the file, please remove the `clib` import and these lines too:

```python
with clib.Session() as _lib:
    gmt_version = Version(_lib.info["version"])
```

Reply:
Done.